### DISCUSSION TITLE
Notification icons in Win 10 taskbar style

### DISCUSSION THREAD

**Discussion by MartinMolhanec** [valinet/ExplorerPatcher#2902] [Q&A]
I am using Win 10 taskbar style but the problem is as follows. All notification icons are hidden and do not respekt setting in Personalisation>Taskbar>Other system tray icons>Hidden icon menu. Is it bug or feature? If I use Win 11 taskbar style all is right.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by MartinMolhanec** [comment]
I have:

 

Edition  Windows 11 Pro

Version 23H2

Installed on        ‎31.‎01.‎2024

OS build               22635.3130

Serial number   S4LS2652

Experience         Windows Feature Experience Pack 1000.22685.1000.0

 

Explorer patcher is: 22621.2861.62.2

 



 

 



 



 



 



 

 



 



 



 



 



 

As you can see it almost default settings.

 

Seems to me that I not using any software which modify UI of Windows.

 

I am using 8GadgetPack, which is older application and TrueLaunchbar, FileMenuTools, FlashFolder and T-Clock Redux. In real the reason that I started using ExplorerPatcher is, that with Windows 10 taskbar style allows using of T-Clock Redux, which is not W11 compatible. And, indeed, T-Clock Redux is working but all notification are hidden. See:

 



 

But I settings:

 



 

Of course, when I switch taskbar style to W11, I see this:

 

 

Which is what I want, but with this setting is not compatible T-Clock Redux ☹

 

Help will be appropriate 😊 

 

In all cases your software is amazing!

 

Best regards,

 

Martin molhanec

 

 

####################################################
Martin Molhanec M.Sc., Ph.D., Assistant Professor
Czech Technical University in Prague, Faculty of Electrical Engineering
Technická 2, 166 27  Prague 6, Czech Republic
####################################################

**Comment by bbmaster123** [comment]
should be as simple as using the GUI to turn that on for the win10 taskbar, which you need to do as it uses a different setting to toggle it on/off

to the GUI > taskbar > customize notification area icons, and check "show all icons and notifications in the taskbar" at the bottom left.
There are other methods as well, but I would just use the EP GUI unless you report back that its still not working

hope that helps!

**Comment by pyrates999** [comment]
Is it this? https://github.com/valinet/ExplorerPatcher/issues/1454

If so, please close this one.

**Comment by MartinMolhanec** [comment]
Hello, than for your response, but….



 

When I click on Always show all icon I really see all icons and it is not what I want, because so many icons ☹

 

If I for example want to set this on the setting window



 

I click on OK, the setup windows is close, but nothing happened ☹and when I open setting window again I see

 



 

So setting to “Show icon and notification” is not happened and return back to default “Only show notification” ☹

 

Seems to me that it is wrong functioning?

 

Best regards,

Martin Molhanec

**Comment by MartinMolhanec** [comment]
Yes, exactly this is the some problem as I describe!

 

Best regards,

Martin Molhanec

**Comment by MartinMolhanec** [comment]
I found a way-around in this answer.

 

TBE4034 <https://github.com/TBE4034>  commented on Jul 28, 2023 <https://github.com/valinet/ExplorerPatcher/issues/1454#issuecomment-1656100362> 


As a work-around, I found that you can drag and drop the tray icons between the shown section on the taskbar and the hidden section within the pull up section.

 

Best regards,

Martin Molhanec

**Comment by MartinMolhanec** [comment]
I found a way-around in this answer.

 

TBE4034 <https://github.com/TBE4034>  commented on Jul 28, 2023 <https://github.com/valinet/ExplorerPatcher/issues/1454#issuecomment-1656100362> 


As a work-around, I found that you can drag and drop the tray icons between the shown section on the taskbar and the hidden section within the pull up section.

 

Best regards,

Martin Molhanec

**Comment by pyrates999** [comment]
Please stop responding via email as it's adding a whole bunch more text and empty lines.

This is also a duplicate of https://github.com/valinet/ExplorerPatcher/issues/1454

Please close this.

**Comment by MartinMolhanec** [comment]
As a work-around, I found that you can drag and drop the tray icons between the shown section on the taskbar and the hidden section within the pull up section.
